src/multihash.cr

```diff
@@ -2,29 +2,75 @@
 require "./hashes"
 
 module Multihash
-  class Multihash
-    @function_code : Int32
+  HEX_SIZE_IN_BYTES = 2
+  struct Multihash
+    @hash_code : Int32
+    @len : Int32
 
-    def initialize(hash_function : String, len : Int32, digest : String)
+    def initialize(hash_name : String, digest : String)
```

Comment:
I think that's a good idea for an additional constructor. I think strings are nicer to work with, so let's keep it represented internally as a hex string.